api/wasm/cpp/proxy_wasm_api.h

```diff
@@ -180,8 +180,8 @@ class GrpcCallHandlerBase {
   void cancel();
 
   virtual void onCreateInitialMetadata() {}
-  virtual void onSuccess(std::unique_ptr<WasmData> message) = 0;
-  virtual void onFailure(GrpcStatus status, std::unique_ptr<WasmData> error_message) = 0;
+  virtual void onSuccess(size_t body_size) = 0;
+  virtual void onFailure(GrpcStatus status) = 0;
```

Comment:
Shouldn't we include `status_code` here as well?

Reply:
You mean on success?  It would always be Ok